{"0.1.1":{"info":{"author":"Guillaume LEFEBVRE","author_email":"guillaume@ldmail.fr","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/Memnoch9178/mem-utils","keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"memnoch-utils-exception","package_url":"https://pypi.org/project/memnoch-utils-exception/","platform":null,"project_url":"https://pypi.org/project/memnoch-utils-exception/","project_urls":{"Homepage":"https://github.com/Memnoch9178/mem-utils"},"provides_extra":null,"release_url":"https://pypi.org/project/memnoch-utils-exception/0.1.1/","requires_dist":["memnoch-utils-texte"],"requires_python":">=3.6","summary":null,"version":"0.1.1","yanked":false,"yanked_reason":null},"last_serial":25704106,"urls":[{"comment_text":"","digests":{"blake2b_256":"3acce8b0a573b0df1f0ab3773a80001460f32002ee819532096c5a75c0eb5696","md5":"5ebdb44e73f0575a5168d4a175188c72","sha256":"2952d7c68e9b0303ca5d743321a4e45a80fde99d8af1f6da7e2cda0bf943c952"},"downloads":-1,"filename":"memnoch_utils_exception-0.1.1-py3-none-any.whl","has_sig":false,"md5_digest":"5ebdb44e73f0575a5168d4a175188c72","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":3753,"upload_time":"2024-10-28T11:27:39","upload_time_iso_8601":"2024-10-28T11:27:39.869193Z","url":"https://files.pythonhosted.org/packages/3a/cc/e8b0a573b0df1f0ab3773a80001460f32002ee819532096c5a75c0eb5696/memnoch_utils_exception-0.1.1-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"56c614ea3c7157f232b9de6e6de3c4fcecbf5513ff4dcdb074b6acf797195ff1","md5":"78457b932fa7ee5c7674dd87afc28ba8","sha256":"e8d80ef020318699c89f2e5341d42df54d535c3724528d023afb3f024d1247fb"},"downloads":-1,"filename":"memnoch_utils_exception-0.1.1.tar.gz","has_sig":false,"md5_digest":"78457b932fa7ee5c7674dd87afc28ba8","packagetype":"sdist","python_version":"source","requires_python":">=3.6","size":3393,"upload_time":"2024-10-28T11:27:41","upload_time_iso_8601":"2024-10-28T11:27:41.502215Z","url":"https://files.pythonhosted.org/packages/56/c6/14ea3c7157f232b9de6e6de3c4fcecbf5513ff4dcdb074b6acf797195ff1/memnoch_utils_exception-0.1.1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}